test/autoupdate.test.ts

```diff
@@ -604,4 +604,77 @@ describe('test `merge`', () => {
 
     expect(scope.isDone()).toEqual(true);
   });
+
+  test('continue if merging throws an error', async () => {
```

Comment:
this is what i wanted to do! i couldn't figure out how to make multiple scopes work